intel_extension_for_transformers/llm/runtime/graph/tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+optimum==1.13.2
+optimum-intel==1.11.0
```

Comment:
If everything needs optimum, we need to add it to our package requirements in `setup.py`. Otherwise we shouldn't require it unnecessarily. See also https://github.com/intel/intel-extension-for-transformers/pull/695#discussion_r1400211576.